sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -370,14 +361,16 @@ def _create_auth(self) -> Union[uamqp_JWTTokenAuth, JWTTokenAuth]:
             update_token=False,
         )
 
-    def _close_connection(self):
-        # type: () -> None
+    def _close_connection(self) -> None:
         self._conn_manager.reset_connection_if_broken()
 
     def _backoff(
-        self, retried_times, last_exception, timeout_time=None, entity_name=None
-    ):
-        # type: (int, Exception, Optional[int], Optional[str]) -> None
+        self,
+        retried_times: int,
+        last_exception: Exception,
+        timeout_time: Optional[int]=None,
```

Comment:
We might have to take a look at this because we typically have timeouts as floats.